src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -2653,6 +2653,9 @@
   <data name="IO_BindHandleFailed" xml:space="preserve">
     <value>BindHandle for ThreadPool failed on this handle.</value>
   </data>
+  <data name="IO_InconsistentLinkType" xml:space="preserve">
+    <value>The link file system type is inconsistent with the link target system type.</value>
```

Comment:
"link target system type"?

Reply:
Pretty sure it should be "link **target** file system type" and it refers to disallow `FileInfo` to target a directory and vice versa.

Reply:
"The link's file-system entry type is inconsistent with that of its target."?

Reply:
I tried to make the message agnostic to "files" or "directories", but apparently I failed. :(

Thanks @mklement0, that suggestion makes more sense. I'll use it.